indra/llimage/llimage.cpp

```diff
@@ -2405,7 +2405,8 @@ void LLImageBase::generateMip(const U8* indata, U8* mipdata, S32 width, S32 heig
 				*(U8*)data = (U8)(((U32)(indata[0]) + indata[1] + indata[in_width] + indata[in_width+1])>>2);
 				break;
 			  default:
-				LL_ERRS() << "generateMmip called with bad num channels" << LL_ENDL;
+				LL_WARNS() << "generateMmip called with bad num channels (" << nchannels << ")" << LL_ENDL;
```

Comment:
Just shouldn't happen. If it did either developer did a mistake developer needs to know about now instead of from logs sometime next year or viewer is likely suffering from memory corruption and should quit and report this issue to bugsplat, not report a warning.